dev/docker/services.env

```diff
@@ -22,8 +22,7 @@ SEARCH_HOST=search
 SEARCH_PORT=9050
 RESTRICTER_URL=http://autoupdate:9012/internal/autoupdate
 
-AUTH_HOST=auth
-AUTH_PORT=9004
+KEYCLOAK_URL=http://keycloak:8080
```

Comment:
Is there a specific reason this has to be `_URL`?
Intuitively I would prefer it being consistent with the other services defining `KEYCLOAK_HOST` and `KEYCLOAK_PORT` variables (if needed also `KEYCLOAK_PROTOCOL` which can be concatenated to `KEYCLOAK_PROTOCOL://KEYCLOAK_HOST:KEYCLOAK_PORT` within the service. As is done everywhere else.

https://github.com/OpenSlides/OpenSlides/blob/main/DEVELOPMENT.md#environment-variables

Reply:
This seems to already be expected here:
https://github.com/OpenSlides/openslides-proxy/pull/18/files#diff-590bd89681945f557a56099d2c2b293d216d845ce1e2709dd652bbed1e199177R199 and
https://github.com/OpenSlides/openslides-proxy/pull/18/files#diff-590bd89681945f557a56099d2c2b293d216d845ce1e2709dd652bbed1e199177R119

Reply:
The endpoint could have a path, so another variable KEYCLOAK_PATH would be needed:
KEYCLOAK_PROTOCOL://KEYCLOAK_HOST:KEYCLOAK_PORT$KEYCLOAK_PATH
... I like KEYCLOAK_URL much better.